cursive/src/backends/curses/n.rs

```diff
@@ -396,19 +396,30 @@ impl backend::Backend for Backend {
     }
 
     fn print_at(&self, pos: Vec2, text: &str) {
-        ncurses::mvaddstr(pos.y as i32, pos.x as i32, text);
+        let text=&text.replace('\0'," "); // or the unwrap() will panic
```

Comment:
In retrospect, replacing `\0` with spaces like this might not be a good idea due to this causing heap allocation which may not be desired considering the arg is `&str`. Maybe finding another way here(and below) would be in order?

Reply:
actually nevermind, because internally [`mvaddstr`](https://github.com/jeaye/ncurses-rs/blob/3aa22bc279e4929e3ab69d49f75a18eda3e431e9/src/lib.rs#L720-L725) does eventually call [`CString::new`](https://doc.rust-lang.org/std/ffi/struct.CString.html#method.new) [here](https://github.com/jeaye/ncurses-rs/blob/3aa22bc279e4929e3ab69d49f75a18eda3e431e9/src/lib.rs#L65) which does allocate memory on the heap. So, it still ends up allocating heap anyway.